gui/README.md

```diff
@@ -6,11 +6,13 @@ The Liana graphical interface.
 
 You will need a few dependencies in order to run correctly this software. For Linux systems, those
 are:
+- [`udev`](https://www.freedesktop.org/software/systemd/man/udev.html) (On Debian/Ubuntu `apt install udev`)
 - [`fontconfig`](https://www.freedesktop.org/wiki/Software/fontconfig/) for access to fonts (On Debian/Ubuntu `apt install libfontconfig1-dev`)
 - [`libudev-dev`](https://www.freedesktop.org/software/systemd/man/libudev.html) to communicate with devices through USB (On Debian/Ubuntu `apt install libudev-dev`)
 
 In addition, if you want to build the project from source, you will need:
 - [`pkg-config`](https://www.freedesktop.org/wiki/Software/pkg-config/) (On Debian/Ubuntu `apt install pkg-config`)
+- `cmake` (On Debian/Ubuntu apt install cmake)
```

Comment:
Why do you think `cmake` and `udev` are necessary dependencies? I strongly suspect they are not since in a fresh Ubuntu container we can build and test the GUI without them:
https://github.com/wizardsardine/liana/blob/29ab7f30b6984249aa83c92dafeb8c2ca8a343c5/.github/workflows/main.yml#L93-L98

Reply:
I got issue at build time on some fresh install ubuntu or debian (proxmox VM)

Reply:
i'm building liana-gui on a fresh Debian12 today, got this:

```shell
~/liana/gui$ cargo build --release 
*   Compiling proc-macro2 v1.0.64
   Compiling quote v1.0.26
   Compiling libc v0.2.141
   Compiling cfg-if v1.0.0
   Compiling pkg-config v0.3.26
error: linker `cc` not found
  |
  = note: No such file or directory (os error 2)

error: could not compile `quote` (build script) due to previous error
warning: build failed, waiting for other jobs to finish...
error: could not compile `proc-macro2` (build script) due to previous error
error: could not compile `libc` (build script) due to previous error
```

fixed by installing build-essential (I guess cmake + udev in build-essential),
should I replace requiring cmake+udev by build-essential?